packages/clerk-auth/package.json

```
@@ -38,16 +38,22 @@
   },
   "homepage": "https://github.com/honojs/middleware",
   "peerDependencies": {
-    "@clerk/backend": ">=0.30.0 <1",
+    "@clerk/backend": "1",
+    "@clerk/shared": "2",
     "hono": ">=3.*"
   },
   "devDependencies": {
-    "@clerk/backend": "^0.30.1",
+    "@clerk/backend": "1",
```

We can change this to:
```suggestion
    "@clerk/backend": "^1.0.0",
    "@clerk/shared": "^2.0.0",
```